person.rb

```diff
@@ -53,10 +53,3 @@ def correct_name
     @nameable.correct_name[0...10]
   end
 end
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227524767-48e92808-47ba-48d4-b299-dec849d8d824.png)

**[OPTIONAL]**

- [ ] It would be great if you add a descriptive summary for your project to make your documentation professional and descriptive 👍